Library/RemoteConfig/RemoteConfigFeature.swift

```diff
@@ -2,13 +2,15 @@ import Foundation
 
 public enum RemoteConfigFeature: String, CaseIterable {
   case consentManagementDialogEnabled = "consent_management_dialog"
+  case creatorDashboardHiddenEnabled = "creator_dashboard_hidden"
```

Comment:
sure thing

Reply:
@msadoon so by removing "hidden" we'll need to have the flag defaulted to `true` so that it reads correctly. Because the intended functionality is to hide the dashboard when then flag is on.

Reply:
Well we're eventually going to completely remove it right? In like a month or so? So its' fine that it's defaulted `true`, because well - it is available in our app - and the "feature" we are building is removing it.

Reply:
ok. will update